tests/src/cgeo/geocaching/files/ProgressInputStreamTest.java

```diff
@@ -9,7 +9,7 @@
 public class ProgressInputStreamTest extends TestCase {
 
     public static void testRead() throws Exception {
-        ProgressInputStream stream = new ProgressInputStream(IOUtils.toInputStream("test"));
+        ProgressInputStream stream = new ProgressInputStream(IOUtils.toInputStream("test", (String) null));
```

Comment:
To extend on the remark of @samueltardieu: It's always bad if tests run green under a German locale, but not a French one, etc. Therefore we should avoid relying on any changeable platform defaults, unless the data being operated on also depends directly on that platform default. In this specific case, I'm okay with your change, since "test" will be encoded fine with any locale.